### DISCUSSION TITLE
Broken on IOS - 1.4.1

### DISCUSSION THREAD

**Discussion by ryanatearth** [niklasvh/html2canvas#2909] [Q&A]
Hi, been using the library a while now and loving it.  Very truly amazing.  However one of the projects I am working on, now requires to be used via Mobile devices.  Using the latest iphone 13 Pro Max and on Safari/Chrome, html2canvas just stops working.  I am using the latest release of 1.4.1.

Basically it will start, and all I will get in the console debug logs is:

`DEBUG #1 1ms Starting document clone with size 428x859 scrolled to 0, -347`

Then nothing happens.  The html2canvas promise never returns.

Is this a known issue and/or a work-around or fix?  Or is somebody possibly looking into this?

Thanks in advance for your help and feedback.  Really looking forward to making this work on all devices.

Cheers!

**Comment by mattyforth** [comment]
I'm also getting the same issue, in iOS and also Safari on my Mac. No other errors/logs after the initial message you are also seeing.

Looks like this is a similar issue: https://github.com/niklasvh/html2canvas/issues/2864

**Reply by ryanatearth** [reply]
Yes looks very similar.  I'm not sure if there are active people working on this or not.  Maybe just in people's spare time?
